run.sh

```diff
@@ -0,0 +1,4 @@
+grunt
```

Comment:
Can change this to `grunt build` so that it won't run the `connect:static` task.